### DISCUSSION TITLE
Audiveris does not start on windows 11

### DISCUSSION THREAD

**Discussion by EMHmark7** [Audiveris/audiveris#770] [Q&A]
I installed it for first time. When I start it, for a fraction of second I see a window, not enough time to really see it, and then nothing. I don't see an app in task manager. 

meanwhile,  in Python the following does not work (File not found):
```
import subprocess

def convert_pdf_to_musicxml(pdf_path, output_dir):
    command = [
        'java', '-jar', 'audiveris.jar', 
        '-batch', 
        '-output', output_dir, 
        pdf_path
    ]
    subprocess.run(command)

convert_pdf_to_musicxml(r"D:\Jupyter\Music_Partitions\afile.pdf", r"D:\Jupyter\Music_Partitions\afile" )
```
Any clue? 

Thanks
Marc

**Comment by fashffffffff** [comment]
Бро, тоже предпроф делаешь?

**Reply by EMHmark7** [reply]
I try to interpret your reply. I need help for understanding your motivation for the question.

**Reply by fashffffffff** [reply]
Sorry, nothing important

**Comment by hbitteur** [comment]
Could you try with the 5.4 version which was just released?
It is supposed to correctly handle the case you described.
